CHANGES.rst

```diff
@@ -1,23 +1,53 @@
 Changelog
 =========
 
-0.6.0
+1.0.0
 -----
-- Support native boolean type
-- **Backward incompatible change:** Columns of SQLAlchemy type String are now created with the HANA SQL type NVARCHAR.
-  The previously used HANA SQL type VARCHAR has been only designed for 7-bit ASCII character data. Storing other
-  non-ASCII characters in a different encoding like UTF-8 was sometimes possible but never intended or recommended.
-  It may cause unexpected behavior for certain database-side operations like sorting or failures with string functions.
+
+Breaking Changes
+~~~~~~~~~~~~~~~~
+- By default native booleans are used. If integer based columns should be used, specify
+  ``use_native_boolean=False`` in ``create_engine``
+- Columns of SQLAlchemy type String are now created with the HANA SQL type NVARCHAR.
+  The previously used HANA SQL type VARCHAR has been only designed for 7-bit ASCII character data.
+  Storing other non-ASCII characters in a different encoding like UTF-8 was sometimes possible but
+  never intended or recommended
+  It may cause unexpected behavior for certain database-side operations like sorting or failures
+  with string functions
   With the introduction of SAP HANA Cloud, the SQL type VARCHAR is also only an alias for NVARCHAR.
-  Please consider a migration from VARCHAR to NVARCHAR.
-- **Backward incompatible change:** Columns of SQLAlchemy type Text are now created with the HANA SQL type NCLOB instead of CLOB.
-  Like the HANA SQL type VARCHAR, CLOB was designed for 7-bit ASCII character data. This change also ensures consistency and
-  compatibility with SAP HANA Cloud, where CLOB is just an alias for NCLOB. Please consider a migration from CLOB to NCLOB.
+- Columns of SQLAlchemy type Text are now created with the HANA SQL type NCLOB instead of CLOB.
+  Like the HANA SQL type VARCHAR, CLOB was designed for 7-bit ASCII character data.
+  This change also ensures consistency and compatibility with SAP HANA Cloud, where CLOB is just an
+  alias for NCLOB
+- Removed ``pyhdb`` support because  ``pyhdb`` is out of maintenance and the GitHub repository was
+  archived. Please migrate to ``hdbcli``
+- Removed support for python versions below version 3.8
+- Removed support for SQLAlchemy below version 1.4
+- Removed support for hdbcli below version 2.10
+- Removed the hidden and outdated feature ``auto_convert_lobs``
+
+Features
+~~~~~~~~
+- Official support for SQLAlchemy 1.4 and 2.0
+- Official support for Python 3.11 and 3.12
+- Added CI with linters and testing utilizing the SQLAchemy and alembic test suite
+- Support the SAP HANA datatype ``SMALLDECIMAL``
+- Support native booleans (this is the new default)
+- sqlalchemy-hana is fully typed and exports its types
+- The alembic dialect left the preview stage and is now included by default
+- Specified the caching support explicitly (the value might change in the future)
```

Comment:
Maybe highlight that we are talking about SQLAlchemy statement caching. How about mentioning the current value/setting?